client/pages/general-project-info.tsx

```diff
@@ -0,0 +1,127 @@
+import { Form, Formik } from 'formik';
+import { Field, Radio, Textarea } from '@components';
+
+export const GeneralProjectInfo: React.FC = () => {
+  return (
+    <Formik initialValues={{}} onSubmit={() => {}}>
+      {({ values }: any) => (
```

Comment:
Let's start working on an interface for our form values, it'll really help us down the road as we start working with these values